manifests/w/Wondershare/Anireel/1.1.2/Wondershare.Anireel.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=NVS1.5-1-22483-1000
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Wondershare.Anireel
+PackageVersion: 1.1.2
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.wondershare.com/cbs_down/anireel_full9589.exe
```

```suggestion
  InstallerUrl: https://download.wondershare.com/cbs_down/anireel_64bit_full9589.exe
```
If you want it to install `1.1.2` then probably use https://download.wondershare.com/cbs_down/anireel_64bit_full9589.exe?

---

From where did you get this?

---

It's the offline installer name from your screenshot: anireel_64bit_full9589.exe